yml/obsloctap.yaml

```diff
@@ -321,3 +321,22 @@ tables:
         ivoa:unit: m
         datatype: char
         votable:arraysize: "*"
+      - name: rot_sky_pos
```

Comment:
I think maybe we should prefix these with `lsst_` to indicate they're non-standard fields?  @gpdf ?

Reply:
i added rubin_ it can not hurt